src/app/content/utils/domUtils.ts

```diff
@@ -1,5 +1,9 @@
 import { HTMLElement } from '@openstax/types/lib.dom';
 
+if (typeof(document) !== 'undefined') {
+  import(/* webpackChunkName: "Node.children" */ 'mdn-polyfills/Node.prototype.children');
+}
```

Comment:
Can this be put in some shared place (it is duplicated in another file above) with a comment explaning it?

Reply:
not duplicated, different polyfill